service.go

```diff
@@ -74,6 +74,7 @@ type Service struct {
 	AlertCreation          string               `json:"alert_creation,omitempty"`
 	AlertGrouping          string               `json:"alert_grouping,omitempty"`
 	AlertGroupingTimeout   *uint                `json:"alert_grouping_timeout,omitempty"`
+	Metadata               interface{}          `json:"metadata,omitempty"`
```

Comment:
Are we able to be more precise than `interface{]`?

Reply:
It doesn't looks like it's any more specific in the IncidentDetails itself: https://github.com/PagerDuty/go-pagerduty/blob/master/webhook.go#L30 . I assumed in all cases they're using this object pretty freeform.

Reply:
@tlianza Thanks for adding this! As a rule we're only adding API features that are documented in the PagerDuty API Docs and the Metadata fields isn't listed. I'll contact that engineering team internally and see if there are plans to document it.

Reply:
@stmcallister it is shown in the example JSON document on this page:

https://developer.pagerduty.com/docs/webhooks/v2-overview/

What's interesting is that there is a `metadata` field documented, but it's shown as part of the `incident` object and not the `service` object and its comment seems service related: "The metadata saved on the service."

It's extremely bizarre that you'll show that the fields exist in the example JSON, but not document them and refuse to support them. Why even show them in the example JSON then?